cedar-policy-validator/src/rbac.rs

```diff
@@ -1149,6 +1150,48 @@ mod test {
         (principal_euid, action_euid, resource_euid, schema)
     }
 
+    fn assert_validate_policy_succeeds(schema: ValidatorSchema, policy: &Template) {
+        let validate = Validator::new(schema);
+        assert!(
+            validate
+                .validate_policy(&policy, ValidationMode::default())
+                .0
+                .next()
+                .is_none(),
+            "Did not expect any validation notes."
```

Comment:
is "notes" the generic term for warnings+errors?

Reply:
I believe "notes" is the historical way to refer to errors... I'll update the terminology.